README.md

```diff
@@ -0,0 +1,5 @@
+# space book
+
+i am a space book
```

Comment:
New Line Should start with a Capital letter `I`", "Add period at the end of the sentence", "Change commit message to `Add book content`